background/src/api/graphql.js

```diff
@@ -1,14 +1,12 @@
 import axios from 'axios'
 
 const MAINNET_ENDPOINTS = [
-    'https://mars2.9cscan.com/graphql',
-    'https://mercury2.9cscan.com/graphql',
-    'https://d131807iozwu1d.cloudfront.net/graphql'
+    'http://localhost:38080/graphql',
```

Comment:
Is it okay to leave it non-configurable at this moment?

Reply:
@dahlia Frankly said, I missed 😅. I'll address that extract configuration on next PR.